net-utils/src/ip_echo_server.rs

```diff
@@ -130,7 +130,7 @@ async fn process_connection(
             .await??;
 
             debug!("Connection established to tcp/{}", *tcp_port);
-            let _ = tcp_stream.shutdown().await;
+            _ = tcp_stream.shutdown().await;
```

Comment:
Sorry, I meant it could probably be just:
```suggestion
           tcp_stream.shutdown().await;
```

Reply:
Ah, well, should we do something with that Result? Add `?`...?

Reply:
Ha, sorry for being so slow! That sounds good to me